scripts/build-xcframework.sh

```diff
@@ -19,8 +21,13 @@ generate_xcframework() {
     for sdk in "${sdks[@]}"; do
         if [[ -n "$(grep "${sdk}" <<< "$ALL_SDKS")" ]]; then
             xcodebuild archive -project Sentry.xcodeproj/ -scheme "$scheme" -configuration Release -sdk "$sdk" -archivePath ./Carthage/archive/${scheme}${sufix}/${sdk}.xcarchive CODE_SIGNING_REQUIRED=NO SKIP_INSTALL=NO CODE_SIGN_IDENTITY= CARTHAGE=YES MACH_O_TYPE=$MACH_O_TYPE
-            
+                 
             createxcframework+="-framework Carthage/archive/${scheme}${sufix}/${sdk}.xcarchive/Products/Library/Frameworks/${scheme}.framework "
+
+            if [ "$MACH_O_TYPE" = "staticlib" ]; then
+                local infoPlist="Carthage/archive/${scheme}${sufix}/${sdk}.xcarchive/Products/Library/Frameworks/${scheme}.framework/Info.plist"
+                plutil -replace "MinimumOSVersion" -string "9999" "$infoPlist"
```

Comment:
`h`: Does that have other implications for Xcode versions that know how to parse the `MinimumOSVersion` correctly?

Reply:
I dont know.

I have tried it for iOS, Catalyst, tvOS.
I validated with the app store for all those above. 
Everything is working with this change.

Reply:
Okay, if you validated it, we should be fine.